terminatorlib/config.py

```diff
@@ -118,6 +118,7 @@
             'title_font'            : 'Sans 9',
             'putty_paste_style'     : False,
             'smart_copy'            : True,
+            'disable_mousewheel_zoom': False,
```

Comment:
I think this should be a profile-level setting, but I'm not sure if it fits more into the Scrolling or the General sections.

Reply:
Yeah agreed, I'll think about it

Reply:
I think "general" might be more suitable.

Every other option in the "Scrolling" section is related to scrolling up and down through the content of a terminal, which is not what this PR is about.